Regression/WarpX-tests.ini

```diff
@@ -53,10 +52,7 @@ branch = 21.12
 [source]
 dir = /home/regtester/AMReX_RegTesting/warpx
 branch = development
```

Comment:
For numbered branches such as AMReX `21.12`, the regression script does not parse them as string and fails:
```
  File "/tmp/ci-m27dcH8Ld2/regression_testing/repo.py", line 38, in get_branch_name
    return self.branch_wanted.strip("\"")
AttributeError: 'float' object has no attribute 'strip'
```

Reply:
Fix in https://github.com/AMReX-Codes/regression_testing/pull/107